### PR TITLE
Website dev

### PR DESCRIPTION
**Pull request by TianyuDu**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#### Session 2 (July 25, Tuesday, 1:00 P.M. - 4:00 P.M. PST)
+- Numpy
+  - What is Numpy and why do we need it?
+  - Arrays
+  - Reshaping, slicing, broadcasting
+- Pandas
+  - Dataframes
+  - Pivots
+- The Python plotting ecosystem
+  - Topic 1 with example time-allocation.
```
Comment (leopoldcambier) [inline index.md:61]: Are we missing something here?

File: raw_syllabus.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# ICME Python Summer Workshop, 2023 edition
```
Comment (leopoldcambier) [inline raw_syllabus.md]: Is this file still needed?

Comment (TianyuDu) [reply raw_syllabus.md]: Do not need the `raw_syllabus.md` anymore, do you want to remove it?

File: index.md (hunk 1 above)
Comment (TianyuDu) [reply index.md:61]: I have added the visualization topic.

Comment (TianyuDu) [reply index.md:61]: I basically copied-pasted things from the syllabus, feel free to add new contents to `index.md` after merging.

File: raw_syllabus.md (hunk 2 above)
Comment (leopoldcambier) [reply raw_syllabus.md]: Yeah let's nuke it.

File: index.md (hunk 1 above)
Comment (leopoldcambier) [reply index.md:61]: Sg.

**Review by leopoldcambier — APPROVED** [review]
